### DISCUSSION TITLE
都3个多月了，已经按要求提交多次资料，退款还没有处理到账，太难了

### DISCUSSION THREAD

**Discussion by ineedyou2008** [getlantern/lantern-forum-cn#888] [General]
管理员，我们已经来往了10多次邮件了。我也按要求提交了退款的资料和退款理由了。距上次联系又已经1个多月过去了，现在没有任何退款进展？？？我的邮箱是275693730@qq.com。期待你的回复，谢谢。

**Comment by MaxMKMax** [comment]
@ineedyou2008 您好. 照道理应该已经给您进行退款了. 我帮您追问一下. 再和您确认. 如果没有消息请您下周一再找我.
